en/mkdocs.yml

```diff
@@ -52,33 +52,39 @@ strict: true
 # Navigation
 nav:
     - Home: index.md
-    - Services:
-          - Service Concepts: services/service-concepts.md
-          - Create Your First Service: services/create-your-first-service.md
-    - APIs:
-          - API Concepts: apis/api-concepts.md
-          - Create and Publish Your First API: apis/create-and-publish-your-first-api.md
-    - Integrations:
-          - Integration Concepts: integrations/integration-concepts.md
-          - Build Your First Integration: integrations/build-your-first-integration.md
-    - Remote Apps:
-          - Connect Remote Applications: remote-apps/connect-remote-applications.md
-    - Observability:
-          - Observability Overview: observability/observability-overview.md
-          - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
-          - Performance Analysis: observability/analyze-performance.md
-    - Insights:
+    - Concepts:
+          - Concepts: concepts/concepts.md
+    - Quick Start Guide:
+          - Quick Start Guide: qsg/quick-start-guide.md
+    - Develop:
+          - Create Project
+                - Create API: develop/create-api.md
+                - Create Integration: develop/create-integration.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
+          - Create API Product: manage/create-api-product.md
+          - Manage Endpoints: manage/manage-endpoints.md
+          - API Documentation: manage/api-documentation.md
```

Comment:
Should the API related ones come here?

Reply:
Removed the ones that do not belong here as per our offline chat.